src/components/transactions/TxDetails/TxData/Transfer/index.tsx

```diff
@@ -31,7 +31,7 @@ const TransferTxInfo = ({ txInfo, txStatus }: TransferTxInfoProps) => {
   return (
     <Box>
       <TransferTxInfoSummary txInfo={txInfo} txStatus={txStatus} />
-      <Box display="flex" alignItems="flex-end">
+      <Box display="flex" alignItems="center">
```

Comment:
Why this change? I suggest leaving it as it was

Reply:
The button is otherwise misaligned:

![image](https://user-images.githubusercontent.com/20442784/191031232-47fd5f74-bc1c-41b1-b8ba-31f4cb9cdf68.png)

Reply:
It looks misaligned on my end without the styling adjustment. I'll revert my changes though and we can look at it separately.